src/renderers/webgl/objects/label/background.ts

```diff
@@ -14,15 +14,15 @@ export class LabelBackground {
   private sprite: Sprite
   private label: Text | BitmapText
   private container: Container
-  private rect: Rectangle
+  private rect: Size
   private _style: LabelBackgroundStyle
 
   constructor(container: Container, label: Text | BitmapText, style: LabelBackgroundStyle) {
     this.label = label
     this.container = container
     this._style = style
 
-    this.rect = { ...this.label.getLocalBounds() }
+    this.rect = { width: this.label.width, height: this.label.height }
```

Comment:
getting very wonky results when calling `label.getBounds(true)` or `label.getLocalBounds()` with `BitmapText` objects - this seems to be the safer route